packages/sakuli-legacy/src/context/sahi/action/common-action/common-actions.spec.ts

```diff
@@ -112,6 +112,76 @@ describe('common-actions', () => {
             );
 
             await expect(result).toEqual('Second');
-        })
+        });
+
+        it('should return true for stabilized DOM within default timeout', async () => {
+            // GIVEN
+            await driver.get(mockHtml(`
+                <div id="root">
+                </div>
+                <script>
+                const elem = document.getElementById("root");
+                const interval = setInterval(() => {
+                    elem.innerHTML = Date.now();
+                }, 5);
+                setTimeout(() => clearInterval(interval), 1000);
+                </script>
+            `));
+
+            // WHEN
+            const start = Date.now();
+            const result = await api._pageIsStable();
+            const duration = Date.now() - start;
+
+            // THEN
+            expect(result).toBeTruthy();
+            expect(duration).toBeLessThan(2_000);
+        });
+        it('should return true if update interval < default check interval', async () => {
+            // GIVEN
+            await driver.get(mockHtml(`
+                <div id="root">
+                </div>
+                <script>
+                const elem = document.getElementById("root");
+                const interval = setInterval(() => {
+                    elem.innerHTML = Date.now();
+                }, 100);
+                setTimeout(() => clearInterval(interval), 1000);
+                </script>
+            `));
+
+            // WHEN
+            const start = Date.now();
+            const result = await api._pageIsStable();
+            const duration = Date.now() - start;
+
+            // THEN
+            expect(result).toBeTruthy();
+            expect(duration).toBeLessThan(2_000);
+        });
```

Comment:
This test looks very similar to the test `should return true for stabilized DOM within default timeout` above. Maybe I'm not seeing it but isn't this test case already covered by the test above?

Reply:
Indeed. Removed it.